lib/pymedphys/docs/README.rst

```diff
@@ -66,86 +55,45 @@ any code that was using the old version of pymedphys. We try to be abreast of
 this by providing details of any breaking changes from one release to the next
 within the `Release Notes`_.
 
+Mailing list
```

Comment:
Let's scrap the link to the mailing list altogether. Instead let's link to the Discourse. It can be set up to be a mailing list from the user's point of view in the notification settings.